src/App.js

```diff
@@ -1,9 +1,34 @@
-function App () {
+import birdData from "./data/birds";
+import bonusItems from "./data/bonusItems";
+import "./App.css";
+import { useState } from "react";
+import Birdcards from "./components/Birdcards";
+import Cart from "./components/Cart";
+import Checkout from "./components/Checkout";
+
+function App() {
+
+  const [birds, setBirds] = useState([]);
+
+  function handleBirdCount(bird){
+    const birdObj = {...bird};
```

Comment:
`bird` is not a piece of state so we don't need to make a copy of it. This function could be a one-liner

`function handleBirdCount(bird){
    setBirds([...birds, bird])
  }
`